compair/static/compair-config.js

```diff
@@ -245,6 +247,20 @@ myApp.factory('RouteResolves',
             var assignmentId = $route.current.params.assignmentId;
             return AnswerResource.userUnsaved({'courseId': courseId, 'assignmentId': assignmentId}).$promise;
         },
+        userLTIs: function() {
+            if (AuthTypesEnabled.lti) {
+                var userId = $route.current.params.userId;
+                return UserLTIUsersResource.get({'id': userId}).$promise;
+            }
+            return {'objects':[]};
```

Comment:
You still need to return a promise or else there will be issues (since promises are expected) 

try `return $q.when({'objects':[]})`